docs/content/_docs/aggregations.html

```diff
@@ -54,30 +58,108 @@ <h3>Size and Extent</h3>
 
 <img style="width: 100%;" src="{{ 'images/size_extent_3.svg' | relative_url }}"></img>
 
-<h3 id="min">
-  Min Aggregation
-  <a class="link-to" href="#min"><span class="glyphicon glyphicon-link"></span></a>
+## AVERAGE
+
+<h3 id="average">
+  Average Aggregation
+  <a class="link-to" href="#average"><span class="glyphicon glyphicon-link"></span></a>
 </h3>
 
 <h5>JSON</h5>
 
 <pre><code class="language-json">
-{"type": "min", "sampling": {"unit": &lt;unit&gt;, "size": &lt;number&gt;, "extent": &lt;number&gt;}}
+{"type": "average", "sampling": {"unit": &lt;unit&gt;, "size": &lt;number&gt;, "extent": &lt;number&gt;}}
```

Comment:
We should update the sampling definition for the aggregations, `size` was changed to `value` a while ago and `extent` isn't always needed. In grafana we set `unit` and `value`, for example `{"type": "average", "sampling": {"unit": "minutes", "value": 2}}`